AdyenComponents/OnlineBanking/OnlineBankingComponent.swift

```diff
@@ -134,9 +134,12 @@ public final class OnlineBankingComponent: PaymentComponent,
         formViewController.append(issuerListPickerItem)
         formViewController.append(FormSpacerItem(numberOfSpaces: 4))
         formViewController.append(termsAndConditionsLabelItem.addingDefaultMargins())
-        formViewController.append(FormSpacerItem())
-        formViewController.append(continueButton)
-        formViewController.append(FormSpacerItem(numberOfSpaces: 2))
+
+        if configuration.showSubmitButton {
+            formViewController.append(FormSpacerItem())
+            formViewController.append(continueButton)
+            formViewController.append(FormSpacerItem(numberOfSpaces: 2))
```

Comment:
maybe we need one of these spacers outside of the `if`, to preserve the same extra space if there is no button?

Reply:
Ah yeah, good catch. I excluded the single spacer from condition. Thanks Eren!